### ISSUE TITLE
!YOU HAVE ANGERED THE PARTY. PREPARE FOR TERMINATION

### ISSUE THREAD

**Issue by ChineseAgent** [EligiaOfficial/Viscon-ProjectC-Groep4#94]
我希望我的蛋蛋被一个 2 米高的维京女士压碎!!!

我希望我的蛋蛋被一个 2 米高的维京女士压碎!!!!!!!

CODE IS NOT UP TO CCP STANDARDS. GOD EMPEROR XI WILL HAVE YOUR NUTSACKS FOR THIS TRANGRESSION.

YOU HAVE 3 DAYS LEFT TO LIVE.

**Comment by FSBAgent** [comment]
я согласен this is horrible code you will be dragged to the gulag for this вы, западные ублюдки!

**Comment by WitteWither** [comment]
Walgelijke huidige toestand van kwaliteit. Dit is de waarom ons land naar de tyfus is gegaan.
